apps/L1E/instagram/instagram-backend/src/models/user.model.ts

```
@@ -1,20 +1,69 @@
-import { Schema, model, models } from 'mongoose';
+import mongoose, { Mongoose, ObjectId, Schema, model, models } from 'mongoose';
```

ESLint

[error] 'Mongoose' is defined but never used.

(no-unused-vars)